spec/spec_helper.rb

```diff
@@ -59,6 +59,10 @@ def photo_fixture_name
   @photo_fixture_name = File.join(File.dirname(__FILE__), "fixtures", "button.png")
 end
 
+def jwks_file_path
+  @jwks_file = "../../spec/fixtures/jwks.json"
```

Comment:
@jhass Is there a better way to include a fixture file for a test? The issue is that I'm fetching this file using https://github.com/diaspora/diaspora/pull/6095/files#diff-a6ce2f8abec91a3b79ce49ef8b9f213fR27